client/src/app/(modules)/practices/(main)/filters-sidebar.tsx

```diff
@@ -212,37 +218,56 @@ export default function FiltersSidebar() {
               }
             >
               Reset all
+              <Reset className="h-5 w-5" />
             </Button>
             <div className="space-y-4">
-              <MultiCombobox
-                id="source"
-                key="source"
-                name="Source"
-                variant="practices"
-                value={filters.sourceName ?? []}
-                options={practicesFiltersOptions.sourceName}
-                onChange={(value) => setFilters({ ...filters, sourceName: value as SourceName[] })}
-              />
-              <MultiCombobox
-                id="country"
-                key="country"
-                name="Country"
-                variant="practices"
-                value={filters.country ?? []}
-                options={practicesFiltersOptions.country}
-                onChange={(value) => setFilters({ ...filters, country: value as number[] })}
-              />
-              <MultiCombobox
-                id="published-year"
-                name="Published on year"
-                variant="practices"
-                value={filters.year ?? []}
-                options={practicesFiltersOptions.year}
-                onChange={(value) => setFilters({ ...filters, year: value as number[] })}
-              />
+              <div>
+                <label htmlFor="source" className="text-sm text-gray-200">
+                  Sources
+                </label>
+                <MultiCombobox
+                  id="source"
+                  key="source"
+                  name="Source"
```

Comment:
I'm suggesting this change so that when a value is selected, we display the name of the filter, “Sources” (in plural).

```suggestion
                  name="Sources"
```